.github/workflows/ci-cd.yml

```
@@ -0,0 +1,323 @@
+      - name: Run security scan
+        uses: snyk/actions/node@master
+        continue-on-error: true
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          args: --severity-threshold=high
+      
+      - name: Run CodeQL analysis
+        uses: github/codeql-action/analyze@v3
+        with:
+          languages: javascript, typescript
```

actionlint (1.7.7)

input "languages" is not defined in action "github/codeql-action/analyze@v3". available inputs are "add-snippets", "category", "check_name", "checkout_path", "cleanup-level", "expect-error", "matrix", "output", "ram", "ref", "sha", "skip-queries", "threads", "token", "upload", "upload-database", "wait-for-processing"

(action)